content/en/serverless/step_functions/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Serverless Monitoring for AWS Step Functions
+kind: documentation
+further_reading:
+- link: "/serverless/step_functions/installation"
+  tag: "Documentation"
+  text: "Install Serverless Monitoring for AWS Step Functions"
+- link: "/serverless/step_functions/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshoot Serverless Monitoring for AWS Step Functions"
+- link: "/integrations/amazon_step_functions/"
+  tag: "Documentation"
+  text: "AWS Step Functions integration"
+- link: "/serverless/aws_lambda/"
+  tag: "Documentation"
+  text: "Serverless Monitoring for AWS Lambda"
+---
+<div class="alert alert-warning">This feature is in public beta. You can provide feedback by emailing <u>serverless-product@datadoghq.com</u> with the subject line <i>[Step Function Beta Feedback]</i>.</div>
+
+{{< img src="serverless/step_functions/overview.png" alt="New Log index" style="width:100%;" >}}
+
+AWS Step Functions is a serverless orchestration service that lets developers create and manage multi-step application workflows. In addition to Datadog's [AWS Step Functions integration][2], which relies on AWS X-Ray, Datadog offers native AWS Step Function monitoring.
+
+Serverless Monitoring for AWS Step Functions enables you to:
+- See the Lambda functions associated with a Step Function
+- View end-to-end traces for a Step Function execution
+- Set monitors on certain log or trace errors
+- Profile your code (Node.JS and Python)
```

Comment:
@sumedham  Maybe it's worth talking about real-time enhanced metrics? people who are used to the integration metrics always want datadog to provide something more real time, because 5 mins delay on a metric often means a 30 mins delay of alert for monitor to have a reliable evaluation window.

Also the enhanced metrics for individual steps are a key distinguisher compared to CW?

In short, although we didn't explicitly write CW and X-Ray as competitors, people would still ask the questions "I already have CW and X-Ray, why do I need this new thing?"

Reply:
Going to change that last one to "Set monitors on certain trace, metric, log error, or anomaly"